More sentence case fixes for labels, etc.

mapk opened the pull request:
## Description
This PR fixes more labels and copy to conform to Sentence case.

## How has this been tested?
Tested locally.

## Types of changes
Non-breaking changes.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. <!-- Check code: `npm run lint`, Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/ -->
- [ ] My code follows the accessibility standards. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/ -->
- [ ] My code has proper inline documentation. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/ -->
- [ ] I've included developer documentation if appropriate. <!-- Handbook: https://developer.wordpress.org/block-editor/ -->
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR. <!-- React Native mobile Gutenberg guidelines: https://github.com/WordPress/gutenberg/blob/master/docs/contributors/native-mobile.md -->.

mapk committed:
More sentence case fixes for labels, etc.

mkaz reviewed (changes requested):
The changes look good. However, you'll need to update the tests to match the updated text. If you look at the details link from the Travis CI and click through each failing test it will show you each test and where it is failing.

mapk committed:
Fixing e2e tests to match changes.

mapk committed:
Updated more e2e tests and some snapshots too.

gziolo commented:
For reference, there is one failing unit test:
https://travis-ci.com/WordPress/gutenberg/jobs/271982464#L393
it has a corresponding snapshot saved which can be updated manually.

There are also 3 failing e2e tests for meta boxes:
https://travis-ci.com/WordPress/gutenberg/jobs/271982469#L792

It's quite possible that `View Post` is passed from PHP as this is something that depends on the post type.

aduth committed:
Edit Post: Update Options Modal snapshot

aduth committed:
E2E Tests: Revert "View Post" back to uppercase

These labels are associated with the registered post type, defined in WordPress

See: https://github.com/WordPress/WordPress/blob/08f53ef906049b61e9effcfc72d9b1906c8eb5ac/wp-includes/post.php#L1695-L1696

mkaz reviewed (approved):
:heavy_check_mark:
